frontend/www/js/omegaup/components/group/Edit.vue

```diff
@@ -8,7 +8,17 @@
     <ul class="nav nav-pills">
       <li class="nav-item" role="presentation">
         <a
-          href="#"
+          :href="`#${selectedTab}`"
```

Comment:
```suggestion
          :href="`#${AvailableTabs.Edit}`"
```

lo que pasaría si se usa `${selectedTab}` es que al momento de cambiar el valor de esa variable, todos los `href` se cambiarían al mismo valor.

Reply:
Listo